controller_manager/src/controller_manager.cpp

```diff
@@ -344,20 +344,20 @@ controller_interface::return_type ControllerManager::switch_controller(
 
   if (strictness == 0) {
     RCLCPP_WARN(
-      get_logger(), "Controller Manager: To switch controllers you need to specify a "
-      "strictness level of controller_manager_msgs::SwitchController::STRICT "
-      "(%d) or ::BEST_EFFORT (%d). Defaulting to ::BEST_EFFORT.",
+      get_logger(), "Controller manager: to switch controllers you need to specify a  "
```

Comment:
CM is a name so I would keep it.
```suggestion
      get_logger(), "Controller Manager: to switch controllers you need to specify a "
```